src/test/java/org/openrewrite/java/jackson/codehaus/CodehausToFasterXMLTest.java

```diff
@@ -330,4 +330,62 @@ void method() {}
           )
         );
     }
+
+    @Test
+    void replaceWithSetConfigCallGeneric() {
```

Comment:
It helps to add the context of tests added as a link to an issue, where possible.
```suggestion
    @Issue("https://github.com/openrewrite/rewrite-jackson/issues/17")
    @Test
    void replaceWithSetConfigCallGeneric() {
```